pom.xml

```diff
@@ -57,13 +57,13 @@
         <unpack.test.overlays.directory>${project.build.directory}/test-classes</unpack.test.overlays.directory>
         <plugin.checkstyle.version>3.1.2</plugin.checkstyle.version>
         <plugin.lifecycle-mapping.version>1.0.0</plugin.lifecycle-mapping.version>
-        <plugin.surefire.version>2.22.2</plugin.surefire.version>
-        <plugin.dependency.version>3.2.0</plugin.dependency.version>
-        <plugin.war.version>3.3.1</plugin.war.version>
+        <plugin.surefire.version>3.0.0-M6</plugin.surefire.version>
+        <plugin.dependency.version>3.3.0</plugin.dependency.version>
+        <plugin.war.version>3.3.2</plugin.war.version>
         <jaxws-ri.version>2.3.0</jaxws-ri.version>
         <jta.version>1.1</jta.version>
         <log4j.version>2.17.2</log4j.version>
-        <ehcache.version>2.10.6</ehcache.version>
+        <ehcache.version>2.10.10.3.24</ehcache.version>
```

Comment:
Sorry for not noting this during my initial review. For the ehcache version change on FINP-8521, KualiCo added an entry to the pom file prefixed with `<!-- The most recent EhCache 2.x releases are not published to Maven Central for some reason. -->`
Do we need to add this to our pom as well or are we getting the correct ehcache jars as it is? Thank you

Reply:
When compiling locally, I was able to get the appropriate EhCache JAR downloaded without adding extra repository handling. I agree that this particular EhCache version is not available in the central Maven repository, as KualiCo's comment indicated; however, based on the info in my local Maven repo, my local machine was able to download the JAR successfully from KualiCo's financials release repository.

Reply:
Thank you for checking this. Since KualiCo had an issue with this, would it make sense to check the dev and test wars to make sure they are using the correct version after the 6/29 upgrade?

Reply:
We could manually check the DEV and TEST WARs for the dependency version if needed, but since we're referencing the newer version explicitly in our POM, I suspect the DEV/TEST WARs would fail to build if the dependency couldn't be found.

Also, since we now seem to be using the same EhCache version as KualiCo, we could look into removing the EhCache dependency setup from our POM in a future user story.